sdk/identity/azure-identity/azure/identity/_credentials/azure_cli.py

```diff
@@ -67,14 +68,18 @@ def get_token(self, *scopes, **kwargs):  # pylint:disable=no-self-use,unused-arg
 def parse_token(output):
     """Parse output of 'az account get-access-token' to an AccessToken.
 
-    In particular, convert the CLI's "expiresOn" value, the string representation of a naive datetime, to epoch seconds.
+    In particular, convert the "expiresOn" value to epoch seconds. This value is a naive local datetime as returned by
+    datetime.fromtimestamp.
     """
     try:
         token = json.loads(output)
-        parsed_expires_on = datetime.strptime(token["expiresOn"], "%Y-%m-%d %H:%M:%S.%f")
-
-        # calculate seconds since the epoch; parsed_expires_on is naive
-        expires_on = (parsed_expires_on - datetime.fromtimestamp(0)).total_seconds()
+        dt = datetime.strptime(token["expiresOn"], "%Y-%m-%d %H:%M:%S.%f")
+        if hasattr(dt, "timestamp"):
+            # Python < 3.3
```

Comment:
Looks like a flipped sign:
```suggestion
            # Python >= 3.3
```